apps/masterbots.ai/lib/helpers/ai-helpers.ts

```
@@ -0,0 +1,373 @@
+    case AIModels.WordWare:
+      return 'WordWare'
+    default:
+      throw new Error('Unsupported model specified')
+  }
+}
+
+// * This function creates the payload for the AI response
+export function createPayload(
+  json: { id: string },
+  messages: { content: string }[],
+  completion: any
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)